sdks/common/runner-api/src/main/proto/beam_runner_api.proto

```diff
@@ -755,9 +755,12 @@ message FunctionSpec {
   // passed as-is.
   string urn = 1;
 
+  // (Deprecated)
```

Comment:
<!--thread_id:cc_131982377_t; commit:204b9ef802875b46714e87aa283bcb753dd44ac0; resolved:1-->
<!--section:context-quote-->
> **vikkyrk** wrote:
> If we are changing the name of this field, might as well remove it rather than deprecating?

<!--section:body-->
For compatibility with the current Dataflow worker, we need this field to be populated. You'll note that e.g. WindowingStrategyTranslation populates both fields.

In the future, I would suggest prefixing the new name with "deprecated_" but no need to go through that for this PR.